staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -607,17 +607,13 @@ func (s *store) GetList(ctx context.Context, key string, opts storage.ListOption
 	if err != nil {
 		return err
 	}
-	recursive := opts.Recursive
-	resourceVersion := opts.ResourceVersion
-	match := opts.ResourceVersionMatch
-	pred := opts.Predicate
-	ctx, span := tracing.Start(ctx, fmt.Sprintf("List(recursive=%v) etcd3", recursive),
+	ctx, span := tracing.Start(ctx, fmt.Sprintf("List(recursive=%v) etcd3", opts.Recursive),
 		attribute.String("audit-id", audit.GetAuditIDTruncated(ctx)),
 		attribute.String("key", key),
-		attribute.String("resourceVersion", resourceVersion),
-		attribute.String("resourceVersionMatch", string(match)),
-		attribute.Int("limit", int(pred.Limit)),
-		attribute.String("continue", pred.Continue))
+		attribute.String("resourceVersion", opts.ResourceVersion),
```

Comment:
Is there any reason (other than aesthetics-ish things) that those are needed?

Reply:
Having local variables copying values from opts make it seem like they are being mutated during execution of the request. It makes sense for variables like limit, which it is doubled during the pagination. 

However other variables like `match`, `resourceVersion` are constant during execution thus is should be clear that they are directly sourced from `opts` and not from local copy.

This will play big role in followup PRs in which I will split `GetList` function into 3 stages. First stage will be preparing the computed variables based on opts, so having clear distinction between mutable and constant variables will make it easier to verify correctness.

Reply:
> However other variables like match, resourceVersion are constant during execution thus is should be clear that they are directly sourced from opts and not from local copy.

Yeah I dig it.

Reply:
SG